specification/cognitiveservices/data-plane/Language/stable/2022-05-01/conversationalanalysisauthoring.json

```diff
@@ -0,0 +1,3060 @@
+    },
+    "ConversationalAnalysisAuthoringConfusionMatrixRow": {
+      "type": "object",
+      "additionalProperties": {
+        "$ref": "#/definitions/ConversationalAnalysisAuthoringConfusionMatrixEntry"
+      },
+      "x-ms-client-name": "ConfusionMatrixRow"
+    },
+    "ConversationalAnalysisAuthoringConversationExportedEntity": {
+      "required": [
+        "category"
+      ],
+      "type": "object",
+      "properties": {
+        "category": {
+          "type": "string"
+        },
+        "compositionSetting": {
+          "$ref": "#/definitions/ConversationalAnalysisAuthoringCompositionSetting"
+        },
+        "list": {
+          "$ref": "#/definitions/ConversationalAnalysisAuthoringExportedEntityList"
+        },
+        "prebuilts": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConversationalAnalysisAuthoringExportedPrebuiltEntity"
+          }
+        }
+      },
+      "x-ms-client-name": "ConversationExportedEntity"
+    },
+    "ConversationalAnalysisAuthoringConversationExportedIntent": {
+      "required": [
+        "category"
+      ],
+      "type": "object",
+      "properties": {
+        "category": {
+          "type": "string"
+        }
+      },
+      "x-ms-client-name": "ConversationExportedIntent"
+    },
+    "ConversationalAnalysisAuthoringConversationExportedProjectAssets": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ConversationalAnalysisAuthoringExportedProjectAssets"
+        }
+      ],
+      "properties": {
+        "intents": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConversationalAnalysisAuthoringConversationExportedIntent"
+          }
+        },
+        "entities": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConversationalAnalysisAuthoringConversationExportedEntity"
+          }
+        },
+        "utterances": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConversationalAnalysisAuthoringConversationExportedUtterance"
+          }
+        }
+      },
+      "x-ms-discriminator-value": "conversation",
+      "x-ms-client-name": "ConversationExportedProjectAssets"
+    },
+    "ConversationalAnalysisAuthoringConversationExportedUtterance": {
+      "required": [
+        "text",
+        "intent"
+      ],
+      "type": "object",
+      "properties": {
+        "entities": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConversationalAnalysisAuthoringExportedUtteranceEntityLabel"
+          }
+        },
+        "text": {
+          "type": "string"
+        },
+        "language": {
+          "type": "string"
+        },
+        "intent": {
+          "type": "string"
+        },
+        "dataset": {
+          "description": "The dataset for this utterance. Allowed values are 'Train' and 'Test'.",
```

Comment:
Why isn't this an enum then? Using PascalCase for acceptable values here also seems to conflict with most/all enums I'm seeing elsewhere in this swagger. Based on other Language swaggers, I recommend using PascalCase.

Reply:
Today we support only these two pre-defined variables. But down the road we'll allow user input as well. So, it'll change from an enum to a user-defined string. That's why it's done this way for now.